core/common/src/IModel.ts

```diff
@@ -127,7 +127,7 @@ export interface SnapshotOpenOptions extends IModelEncryptionProps, OpenDbKey {
   autoUploadBlocks?: boolean;
 }
 
-/** Options supplied to open an existing StandaloneDb.
+/** Options supplied to open a StandaloneDb.

```

Comment:
Can you turn `StandaloneDb` into a link - `[StandaloneDb]($backend)`?

Reply:
done